pages/common/gox.md

```diff
@@ -0,0 +1,19 @@
+# gox
+
+> A tool for Go cross compilation.
+
+- Compile current Go directory for all OS and arch:
+
+`gox`
+
+- Compile from URL:
+
+`gox {{url_1}} {{url_2}}`
+
+- Compile current directory for `{{os}}`:
+
+`gox -os="{{os}}"`
```

Comment:
Are you missing an extra dash there?

Reply:
Negative. As per the `gox --help` a single dash is listed, but both seem to play nice. I added single dashes because that's how it's displayed in the help.